### PR TITLE
Updated CUID

### PR DESCRIPTION
**Pull request by IDCalTech**
Updated 7 deployment files

### CODE REVIEW

**Review by cdobs — APPROVED** [review]
Everything looks good!

File: deployment/GA01SUMO_Deploy.csv [hunk 1]
```diff
@@ -1,150 +1,150 @@
...
+NBP1609,,AT39-03,,GA01SUMO-SBD12-00-DCLENG000,3,1,2016-10-27T01:47:00,2018-01-14T10:24:00,CGMGA-01SUMO-00003,,CGCON-BDCL12-50009,-42.9204,-42.44093167,,0,5175,
+NBP1609,,AT39-03,,GA01SUMO-SBD12-01-OPTAAD000,3,1,2016-10-27T01:47:00,2018-01-14T10:24:00,CGMGA-01SUMO-00003,,CGINS-OPTAAD-00187,-42.9204,-42.44093167,,1,5175,
+NBP1609,,AT39-03,,GA01SUMO-SBD12-02-FLORTD000,3,1,2016-10-27T01:47:00,2018-01-14T10:24:00,CGMGA-01SUMO-00003,,CGINS-FLORTD-01105,-42.9204,-42.44093167,,1,5175,
+NBP1609,,AT39-03,,GA01SUMO-SBD12-03-HYDGN0000,3,1,2016-10-27T01:47:00,2018-01-14T10:24:00,CGMGA-01SUMO-00003,,CGINS-HYDGN0-49881,-42.9204,-42.44093167,,0,5175,
+NBP1609,,AT39-03,,GA01SUMO-SBD12-04-PCO2AA000,3,1,2016-10-27T01:47:00,2018-01-14T10:24:00,CGMGA-01SUMO-00003,,CGINS-PCO2AA-34231,-42.9204,-42.44093167,,0,5175,
+NBP1609,,AT39-03,,GA01SUMO-SBD12-05-WAVSSA000,3,1,2016-10-27T01:47:00,2018-01-14T10:24:00,CGMGA-01SUMO-00003,,CGINS-WAVSSA-05981,-42.9204,-42.44093167,,0,5175,
+NBP1609,,AT39-03,,GA01SUMO-SBD12-06-METBKA000,3,1,2016-10-27T01:47:00,2018-01-14T10:24:00,CGMGA-01SUMO-00003,,CGINS-METLGR-00034,-42.9204,-42.44093167,,0,5175,
```
[the remark this answers is not in the document]
Comment (snwhite) [reply deployment/GA01SUMO_Deploy.csv:150]: Looks good.

File: deployment/GA05MOAS-GL493_Deploy.csv [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
-AT-26-30,,NBP-15-10,,GA05MOAS-GL493-02-DOSTAM000,1,1,2015-03-15T02:00:00,2015-11-13T20:51:00,CGVGA-05MOAS-00000,CGVEH-GLDROO-00493,CGINS-DOSTAM-00347,-42.64838,-42.499483,,1000,1000,
-AT-26-30,,NBP-15-10,,GA05MOAS-GL493-04-CTDGVM000,1,1,2015-03-15T02:00:00,2015-11-13T20:51:00,CGVGA-05MOAS-00000,CGVEH-GLDROO-00493,CGINS-CTDGVM-09211,-42.64838,-42.499483,,1000,1000,
\ No newline at end of file
+AT26-30,,NBP1510,,GA05MOAS-GL493-00-ENG000000,1,1,2015-03-15T02:00:00,2015-11-13T20:51:00,CGVGA-05MOAS-00000,CGVEH-GLDROO-00493,CGCON-GLDROO-00493,-42.64838,-42.499483,,1000,1000,
+AT26-30,,NBP1510,,GA05MOAS-GL493-01-FLORDM000,1,1,2015-03-15T02:00:00,2015-11-13T20:51:00,CGVGA-05MOAS-00000,CGVEH-GLDROO-00493,CGINS-FLORTM-03553,-42.64838,-42.499483,,1000,1000,
+AT26-30,,NBP1510,,GA05MOAS-GL493-02-DOSTAM000,1,1,2015-03-15T02:00:00,2015-11-13T20:51:00,CGVGA-05MOAS-00000,CGVEH-GLDROO-00493,CGINS-DOSTAM-00347,-42.64838,-42.499483,,1000,1000,
+AT26-30,,NBP1510,,GA05MOAS-GL493-04-CTDGVM000,1,1,2015-03-15T02:00:00,2015-11-13T20:51:00,CGVGA-05MOAS-00000,CGVEH-GLDROO-00493,CGINS-CTDGVM-09211,-42.64838,-42.499483,,1000,1000,
```
[the remark this answers is not in the document]
Comment (snwhite) [reply deployment/GA05MOAS-GL493_Deploy.csv:5]: Looks good.

File: deployment/GA05MOAS-GL494_Deploy.csv [hunk 3]
```diff
@@ -1,5 +1,5 @@
...
-AT-26-30,,UNRECOVERED,,GA05MOAS-GL494-04-CTDGVM000,1,1,2015-03-14T11:30:00,2015-05-06T19:35:00,CGVGA-05MOAS-00000,CGVEH-GLDROO-00494,CGINS-CTDGVM-09214,-42.74585,-42.53177,,1000,1000,Glider lost at sea
\ No newline at end of file
+AT26-30,,UNRECOVERED,,GA05MOAS-GL494-00-ENG000000,1,1,2015-03-14T11:30:00,2015-05-06T19:35:00,CGVGA-05MOAS-00000,CGVEH-GLDROO-00494,CGCON-GLDROO-00494,-42.74585,-42.53177,,1000,1000,Glider lost at sea
+AT26-30,,UNRECOVERED,,GA05MOAS-GL494-01-FLORDM000,1,1,2015-03-14T11:30:00,2015-05-06T19:35:00,CGVGA-05MOAS-00000,CGVEH-GLDROO-00494,CGINS-FLORTM-03554,-42.74585,-42.53177,,1000,1000,Glider lost at sea
+AT26-30,,UNRECOVERED,,GA05MOAS-GL494-02-DOSTAM000,1,1,2015-03-14T11:30:00,2015-05-06T19:35:00,CGVGA-05MOAS-00000,CGVEH-GLDROO-00494,CGINS-DOSTAM-00348,-42.74585,-42.53177,,1000,1000,Glider lost at sea
+AT26-30,,UNRECOVERED,,GA05MOAS-GL494-04-CTDGVM000,1,1,2015-03-14T11:30:00,2015-05-06T19:35:00,CGVGA-05MOAS-00000,CGVEH-GLDROO-00494,CGINS-CTDGVM-09214,-42.74585,-42.53177,,1000,1000,Glider lost at sea
```
[the remark this answers is not in the document]
Comment (snwhite) [reply deployment/GA05MOAS-GL494_Deploy.csv:5]: Looks good.

File: deployment/GA05MOAS-GL495_Deploy.csv [hunk 4]
```diff
@@ -1,5 +1,5 @@
...
-AT-26-30,,AT-26-30,,GA05MOAS-GL495-02-DOSTAM000,1,1,2015-03-14T02:10:00,2015-03-14T23:50:00,CGVGA-05MOAS-00000,CGVEH-GLDROO-00495,CGINS-DOSTAM-00325,-42.74303,-42.514367,,1000,1000,
-AT-26-30,,AT-26-30,,GA05MOAS-GL495-04-CTDGVM000,1,1,2015-03-14T02:10:00,2015-03-14T23:50:00,CGVGA-05MOAS-00000,CGVEH-GLDROO-00495,CGINS-CTDGVM-09209,-42.74303,-42.514367,,1000,1000,
\ No newline at end of file
+AT26-30,,AT26-30,,GA05MOAS-GL495-00-ENG000000,1,1,2015-03-14T02:10:00,2015-03-14T23:50:00,CGVGA-05MOAS-00000,CGVEH-GLDROO-00495,CGCON-GLDROO-00495,-42.74303,-42.514367,,1000,1000,
+AT26-30,,AT26-30,,GA05MOAS-GL495-01-FLORDM000,1,1,2015-03-14T02:10:00,2015-03-14T23:50:00,CGVGA-05MOAS-00000,CGVEH-GLDROO-00495,CGINS-FLORDM-03578,-42.74303,-42.514367,,1000,1000,
+AT26-30,,AT26-30,,GA05MOAS-GL495-02-DOSTAM000,1,1,2015-03-14T02:10:00,2015-03-14T23:50:00,CGVGA-05MOAS-00000,CGVEH-GLDROO-00495,CGINS-DOSTAM-00325,-42.74303,-42.514367,,1000,1000,
+AT26-30,,AT26-30,,GA05MOAS-GL495-04-CTDGVM000,1,1,2015-03-14T02:10:00,2015-03-14T23:50:00,CGVGA-05MOAS-00000,CGVEH-GLDROO-00495,CGINS-CTDGVM-09209,-42.74303,-42.514367,,1000,1000,
```
[the remark this answers is not in the document]
Comment (snwhite) [reply deployment/GA05MOAS-GL495_Deploy.csv:5]: Looks good.

File: deployment/GA05MOAS-GL496_Deploy.csv [hunk 5]
```diff
@@ -1,5 +1,5 @@
...
-AT-26-30,,UNRECOVERED,,GA05MOAS-GL496-04-CTDGVM000,1,1,2015-03-14T11:30:00,2016-03-16T15:06:00,CGVGA-05MOAS-00000,CGVEH-GLDROO-00496,CGINS-CTDGVM-09207,-42.747067,-42.52835,,1000,1000,Glider lost at sea
\ No newline at end of file
+AT26-30,,UNRECOVERED,,GA05MOAS-GL496-00-ENG000000,1,1,2015-03-14T11:30:00,2016-03-16T15:06:00,CGVGA-05MOAS-00000,CGVEH-GLDROO-00496,CGCON-GLDROO-00496,-42.747067,-42.52835,,1000,1000,Glider lost at sea
+AT26-30,,UNRECOVERED,,GA05MOAS-GL496-01-FLORDM000,1,1,2015-03-14T11:30:00,2016-03-16T15:06:00,CGVGA-05MOAS-00000,CGVEH-GLDROO-00496,CGINS-FLORDM-03577,-42.747067,-42.52835,,1000,1000,Glider lost at sea
+AT26-30,,UNRECOVERED,,GA05MOAS-GL496-02-DOSTAM000,1,1,2015-03-14T11:30:00,2016-03-16T15:06:00,CGVGA-05MOAS-00000,CGVEH-GLDROO-00496,CGINS-DOSTAM-00337,-42.747067,-42.52835,,1000,1000,Glider lost at sea
+AT26-30,,UNRECOVERED,,GA05MOAS-GL496-04-CTDGVM000,1,1,2015-03-14T11:30:00,2016-03-16T15:06:00,CGVGA-05MOAS-00000,CGVEH-GLDROO-00496,CGINS-CTDGVM-09207,-42.747067,-42.52835,,1000,1000,Glider lost at sea
```
Comment (snwhite) [inline deployment/GA05MOAS-GL496_Deploy.csv:5]: Looks good.

File: deployment/GA05MOAS-GL538_Deploy.csv [hunk 6]
```diff
@@ -1,5 +1,5 @@
...
-NBP-15-10,,NBP16-09,,GA05MOAS-GL538-02-DOSTAM000,1,1,2015-11-24T17:30:00,2016-11-03T21:36:00,CGVGA-05MOAS-00000,CGVEH-GLDROO-00538,CGINS-DOSTAM-00398,-42.585,-42.66667,,1000,1000,
-NBP-15-10,,NBP16-09,,GA05MOAS-GL538-04-CTDGVM000,1,1,2015-11-24T17:30:00,2016-11-03T21:36:00,CGVGA-05MOAS-00000,CGVEH-GLDROO-00538,CGINS-CTDGVM-09238,-42.585,-42.66667,,1000,1000,
\ No newline at end of file
+NBP1510,,NBP1609,,GA05MOAS-GL538-00-ENG000000,1,1,2015-11-24T17:30:00,2016-11-03T21:36:00,CGVGA-05MOAS-00000,CGVEH-GLDROO-00538,CGCON-GLDROO-00538,-42.585,-42.66667,,1000,1000,
+NBP1510,,NBP1609,,GA05MOAS-GL538-01-FLORDM000,1,1,2015-11-24T17:30:00,2016-11-03T21:36:00,CGVGA-05MOAS-00000,CGVEH-GLDROO-00538,CGINS-FLORDM-03583,-42.585,-42.66667,,1000,1000,
+NBP1510,,NBP1609,,GA05MOAS-GL538-02-DOSTAM000,1,1,2015-11-24T17:30:00,2016-11-03T21:36:00,CGVGA-05MOAS-00000,CGVEH-GLDROO-00538,CGINS-DOSTAM-00398,-42.585,-42.66667,,1000,1000,
+NBP1510,,NBP1609,,GA05MOAS-GL538-04-CTDGVM000,1,1,2015-11-24T17:30:00,2016-11-03T21:36:00,CGVGA-05MOAS-00000,CGVEH-GLDROO-00538,CGINS-CTDGVM-09238,-42.585,-42.66667,,1000,1000,
```
[the remark this answers is not in the document]
Comment (snwhite) [reply deployment/GA05MOAS-GL538_Deploy.csv:5]: Looks good.

File: deployment/GA05MOAS-PG562_Deploy.csv [hunk 7]
```diff
@@ -1,8 +1,8 @@
...
+NBP1510,,UNRECOVERED,,GA05MOAS-PG562-01-CTDGVM000,1,1,2015-11-24T21:30:00,2017-01-27T08:00:00,CGVGA-05MOAS-00000,CGVEH-GLDRGP-00562,CGINS-CTDGVM-09284,-42.57167,-42.65667,,200,200,Glider lost at sea
+NBP1510,,UNRECOVERED,,GA05MOAS-PG562-02-DOSTAM000,1,1,2015-11-24T21:30:00,2017-01-27T08:00:00,CGVGA-05MOAS-00000,CGVEH-GLDRGP-00562,CGINS-DOSTAM-00420,-42.57167,-42.65667,,200,200,Glider lost at sea
+NBP1510,,UNRECOVERED,,GA05MOAS-PG562-03-FLORTM000,1,1,2015-11-24T21:30:00,2017-01-27T08:00:00,CGVGA-05MOAS-00000,CGVEH-GLDRGP-00562,CGINS-FLORTM-03783,-42.57167,-42.65667,,200,200,Glider lost at sea
+NBP1510,,UNRECOVERED,,GA05MOAS-PG562-04-FLORTO000,1,1,2015-11-24T21:30:00,2017-01-27T08:00:00,CGVGA-05MOAS-00000,CGVEH-GLDRGP-00562,CGINS-FLORTO-01329,-42.57167,-42.65667,,200,200,Glider lost at sea
+NBP1510,,UNRECOVERED,,GA05MOAS-PG562-05-NUTNRM000,1,1,2015-11-24T21:30:00,2017-01-27T08:00:00,CGVGA-05MOAS-00000,CGVEH-GLDRGP-00562,CGINS-NUTNRM-00659,-42.57167,-42.65667,,200,200,Glider lost at sea
+NBP1510,,UNRECOVERED,,GA05MOAS-PG562-06-PARADM000,1,1,2015-11-24T21:30:00,2017-01-27T08:00:00,CGVGA-05MOAS-00000,CGVEH-GLDRGP-00562,CGINS-PARADM-50197,-42.57167,-42.65667,,200,200,Glider lost at sea
```
[the remark this answers is not in the document]
Comment (snwhite) [reply deployment/GA05MOAS-PG562_Deploy.csv:8]: Looks good.

File: deployment/GA05MOAS-PG563_Deploy.csv [hunk 8]
```diff
@@ -1,8 +1,8 @@
...
+NBP1510,,JCR15006,,GA05MOAS-PG563-01-CTDGVM000,1,1,2015-11-24T21:47:00,2016-05-02T11:05:00,CGVGA-05MOAS-00000,CGVEH-GLDRGP-00563,CGINS-CTDGVM-09285,-42.58833,-42.645,,200,200,
+NBP1510,,JCR15006,,GA05MOAS-PG563-02-DOSTAM000,1,1,2015-11-24T21:47:00,2016-05-02T11:05:00,CGVGA-05MOAS-00000,CGVEH-GLDRGP-00563,CGINS-DOSTAM-00423,-42.58833,-42.645,,200,200,
+NBP1510,,JCR15006,,GA05MOAS-PG563-03-FLORTM000,1,1,2015-11-24T21:47:00,2016-05-02T11:05:00,CGVGA-05MOAS-00000,CGVEH-GLDRGP-00563,CGINS-FLORTM-03813,-42.58833,-42.645,,200,200,
+NBP1510,,JCR15006,,GA05MOAS-PG563-04-FLORTO000,1,1,2015-11-24T21:47:00,2016-05-02T11:05:00,CGVGA-05MOAS-00000,CGVEH-GLDRGP-00563,CGINS-FLORTO-01330,-42.58833,-42.645,,200,200,
+NBP1510,,JCR15006,,GA05MOAS-PG563-05-NUTNRM000,1,1,2015-11-24T21:47:00,2016-05-02T11:05:00,CGVGA-05MOAS-00000,CGVEH-GLDRGP-00563,CGINS-NUTNRM-00658,-42.58833,-42.645,,200,200,
+NBP1510,,JCR15006,,GA05MOAS-PG563-06-PARADM000,1,1,2015-11-24T21:47:00,2016-05-02T11:05:00,CGVGA-05MOAS-00000,CGVEH-GLDRGP-00563,CGINS-PARADM-50198,-42.58833,-42.645,,200,200,
```
[the remark this answers is not in the document]
Comment (snwhite) [reply deployment/GA05MOAS-PG563_Deploy.csv:8]: Looks good.

**Review by snwhite — APPROVED** [review]
All look good.

**Pull request merged**